README.md

```
@@ -2,10 +2,15 @@
 
 > **Local, Offline Documentation Context Server (MCP) for LLMs & AI Coding Agents**
 
+[![Status: Production Ready (MVP)](https://img.shields.io/badge/Status-MVP%20Ready-brightgreen.svg)](#-mvp-status--readiness)
```

markdownlint-cli2 (0.23.2)

[warning] Link fragments should be valid

(MD051, link-fragments)